projects/angular/src/layout/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,34 @@
+<div role="list" class="clr-breadcrumb-menu">
+  <ng-container *ngIf="items?.length">
+    <clr-breadcrumb-item *ngIf="items.length > max && !isExpanded">
+      <span (click)="expand()" (keydown)="expand()" class="clr-breadcrumb-expand">
+        <cds-icon shape="ellipsis-horizontal"></cds-icon>
+      </span>
```

Comment:
```suggestion
      <span (click)="expand()" (keydown)="expand()" class="clr-breadcrumb-expand">
        <cds-icon shape="ellipsis-horizontal"></cds-icon>
      </span>
```

This element does not have `role="button"` and it should. Please try to use a native `button` element if possible (and remove the custom cursor style).

Reply:
Changed it to clarity link button with some of the styles overridden to match the figma design.